src/data/sql/brk/kot.add_relatie_g_perceel.sql

```diff
@@ -149,4 +146,90 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
-SELECT kot_derive_g_percelen();
\ No newline at end of file
+SELECT kot_derive_g_percelen();
+
```

Comment:
Below is the new SQL. Functionality above is the previous functionality. Modifications caused a slow running query. New function below solves the same problem, but much faster.